lib/iris/tests/unit/fileformats/um/optimal_array_structuring/test_optimal_array_structure.py

```diff
@@ -95,7 +95,13 @@ def test_2d(self):
             ("b", np.array([7, 8, 9, 7, 8, 9])),
         ]
         shape, primaries, elems_and_dims = optimal_array_structure(elements)
-        self.assertEqual(shape, (2, 3,))
+        self.assertEqual(
+            shape,
+            (
+                2,
+                3,
+            ),
+        )
```

Comment:
This formatting looks really horrible, is less readable and seems gratuitous. Is there a configuration we can use to make Black less zealous?